data/categories/sg_sporting_goods.yml

```diff
@@ -12382,24 +12382,126 @@
   children:
   - sg-4-15-1
   - sg-4-15-2
+  - sg-4-15-3
   attributes:
   - age_group
   - color
   - pattern
 - id: sg-4-15-1
   name: E-Scooters
+  children:
+  - sg-4-15-1-1
+  - sg-4-15-1-2
+  - sg-4-15-1-3
+  - sg-4-15-1-4
+  attributes:
+  - age_group
+  - color
+  - pattern
+- id: sg-4-15-1-1
+  name: Commuter E-Scooters
   children: []
   attributes:
   - age_group
   - color
+  - features
+  - pattern
+  - range_type
+  - wheel_type
+- id: sg-4-15-1-2
+  name: Off-Road E-Scooters
+  children: []
+  attributes:
+  - age_group
+  - color
+  - pattern
+  - features
+  - range_type
```

Comment:
I would prioritise a measurement attribute for range:

![image](https://github.com/user-attachments/assets/79b52b55-6f96-4ec6-8355-96f931e7b00c)